server/src/test/java/org/opensearch/index/engine/InternalEngineTests.java

```diff
@@ -6174,8 +6174,8 @@ public void testHistoryBasedOnSource() throws Exception {
                 }
             }
             MapperService mapperService = createMapperService("test");
-            List<Translog.Operation> luceneOps = readAllOperationsBasedOnSource(engine, Engine.HistorySource.INDEX, mapperService);
-            List<Translog.Operation> translogOps = readAllOperationsBasedOnSource(engine, Engine.HistorySource.TRANSLOG, mapperService);
+            List<Translog.Operation> luceneOps = readAllOperationsBasedOnSource(engine, mapperService);
```

Comment:
`luceneOps` and `translogOps` use the same method call, should be always identical? `translogOps` could be dropped I think

Reply:
good catch!